.gitlab-ci.yml

```
@@ -84,6 +85,16 @@ default:
     #    update, it also runs all the nightly checks.
     - if: $CI_PIPELINE_SOURCE == "pipeline"
 
+#### stage:                        canary
+
+canary:
+  stage:                           canary
+  image:                           alpine:3.13
+  script:
+    - "true"
+  tags:
+    - linux-docker
```

```suggestion
    - kubernetes-parity-build
```
to not to queue up our concurrent runners, even the empty job takes ~20 seconds.